src/dispatcher/index.ts

```diff
@@ -12,6 +12,14 @@ export interface MetaPayload {
   content: string;
 }
 
+const applyTemplate = (title: string, template?: string) => {
```

Comment:
This could just be `template ? x : title`

Reply:
Not quite: it has to be `title && template ? template.replace(/%s/g, title) : title;` because otherwise an empty title [would leave the naked template behind](https://github.com/JoviDeCroock/hooked-head/pull/6/files#diff-d437e9d7a32309df382898f68f8381e5R108-R113).